docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -137,68 +136,146 @@ To output predicted class probabilities instead of predicted classes, you can us
 
 ```{.python .input}
 class_probs = predictor.predict_proba(datapoint)
-print(class_probs)
+print(pd.DataFrame(class_probs, columns=predictor.class_labels))
 ```
 
 By default, `predict()` and `predict_proba()` will utilize the model that AutoGluon thinks is most accurate, which is usually an ensemble of many individual models.
-We can instead specify a particular model to use for predictions (e.g. to reduce inference latency).  Before deciding which model to use, let's evaluate all of the models AutoGluon has previously trained using our validation dataset:
+We can instead specify a particular model to use for predictions (e.g. to reduce inference latency).  Before deciding which model to use, let's evaluate all of the models AutoGluon has previously trained using our test data:
 
 ```{.python .input}
-results = predictor.leaderboard(val_data)
+results = predictor.leaderboard(test_data)
```

Comment:
consider adding `extra_info=True` here since this is the indepth tutorial, and mentioning briefly what extra_info does.

Reply:
Added in separate 2nd call, since leaderboard appears a bit overwhelming at first with all this extra-info.